api/v1beta1/openlibertyapplication_types.go

```diff
@@ -441,6 +441,7 @@ const (
 // +kubebuilder:printcolumn:name="Message",type="string",JSONPath=".status.conditions[?(@.type=='Reconciled')].message",priority=1,description="Failure message from reconcile condition"
 // +kubebuilder:printcolumn:name="DependenciesSatisfied",type="string",JSONPath=".status.conditions[?(@.type=='DependenciesSatisfied')].status",priority=1,description="Status of the application dependencies"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",priority=0,description="Age of the resource"
+//+operator-sdk:csv:customresourcedefinitions:displayName="Open Liberty Application",resources={{Deployment,v1,},{Service,v1,},{StatefulSet,v1,},{Route,v1,},{HorizontalPodAutoscaler,v1,}}
```

Comment:
Let's add ServiceAccount and Secret to the list. Both have v1 as the version.
Minor thing, but it'll be good to remove the `,` after v1 to improve readability. 

Let's change to use the CR name as display name: OpenLibertyApplication - same for trace and dump CRs